### PR TITLE
Add promotion banner to promote web app in

### PR DESCRIPTION
**Pull request by thayannevls**
#### What is the purpose of this pull request?
Add `PromotionBanner` to `orderPlaced` to promote the installation of the web app and show prompt.

#### What problem is this solving?
In the store settings, you can choose to show the [install prompt](https://developers.google.com/web/fundamentals/app-install-banners/promoting-install-mobile) when the first purchase is maded. 

#### How should this be manually tested?
[Workspace](https://thay--storecomponents.myvtex.com/checkout/orderPlaced/?og=949500478939)

#### Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Requires change to documentation, which has been updated accordingly.

### CODE REVIEW

File: react/components/OrderInfo/index.tsx [hunk 1]
```diff
@@ -38,7 +38,7 @@ const OrderInfo: FunctionComponent<Props> = ({
 
   return (
     <section>
-      <div className="mv6 w-80-ns w-90 center">
+      <div>
```
Comment (salesfelipe) [inline react/components/OrderInfo/index.tsx]: Do you really need this div?

File: manifest.json [hunk 2]
```diff
@@ -21,7 +23,9 @@
     "vtex.shipping-estimate-translator": "2.x",
     "vtex.address-form": "4.x",
     "vtex.profile-form": "3.x",
-    "vtex.pixel-manager": "1.x"
+    "vtex.pixel-manager": "1.x",
+    "vtex.pwa-components": "0.x",
+    "vtex.store-resources": "0.x"
```
Comment (salesfelipe) [inline manifest.json]: Way don't you bump this bad-boys versions? :)

https://semver.org/#spec-item-4

File: react/components/OrderInfo/index.tsx (hunk 1 above)
Comment (thayannevls) [reply react/components/OrderInfo/index.tsx]: Since I moved the css, I don't need this div. Can I remove it ?

File: manifest.json (hunk 2 above)
Comment (thayannevls) [reply manifest.json]: @salesfelipe I can bump pwa-components to be **1.x**, but store-resources I don't think that I can :eyes:

File: react/components/OrderInfo/index.tsx (hunk 1 above)
Comment (salesfelipe) [reply react/components/OrderInfo/index.tsx]: You tell me...

File: manifest.json (hunk 2 above)
Comment (salesfelipe) [reply manifest.json]: This is just a thing to keep in mind, isn't a must...

**Pull request merged**